src/rp2040/hardware_regs/include/hardware/regs/intctrl.h

```diff
@@ -71,6 +71,12 @@ typedef enum irq_num_rp2040 {
     I2C0_IRQ = 23, ///< Select I2C0's IRQ output
     I2C1_IRQ = 24, ///< Select I2C1's IRQ output
     RTC_IRQ = 25, ///< Select RTC's IRQ output
+    SPARE_IRQ_0 = 26, ///< Select SPARE IRQ 0
```

Comment:
Comparing this to https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2350/hardware_regs/include/hardware/regs/intctrl.h , do these spare IRQs also need to be added to the `#ifdef __ASSEMBLER__` section?

Reply:
Hmmm, just noticed that https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2350/hardware_regs/include/hardware/regs/intctrl.h also has:
```c
#define PLL_USB_IRQ 43
#define POWMAN_IRQ_POW 44
#define POWMAN_IRQ_TIMER 45
#define SPAREIRQ_IRQ_0 46
#define SPAREIRQ_IRQ_1 47
```
and
```c
    PLL_USB_IRQ = 43, ///< Select PLL_USB's IRQ output
    POWMAN_IRQ_POW = 44, ///< Select POWMAN's IRQ_POW output
    POWMAN_IRQ_TIMER = 45, ///< Select POWMAN's IRQ_TIMER output
    SPARE_IRQ_0 = 46, ///< Select SPARE IRQ 0
    SPARE_IRQ_1 = 47, ///< Select SPARE IRQ 1
```
with the naming of `SPAREIRQ_IRQ_0` vs. `SPARE_IRQ_0` seeming to be a bit inconsistent? :thinking:

Reply:
wouldn't it fail to compile if they're the same?

Reply:
> Hmmm, just noticed that https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2350/hardware_regs/include/hardware/regs/intctrl.h also has:
> 
> ```c
> #define PLL_USB_IRQ 43
> #define POWMAN_IRQ_POW 44
> #define POWMAN_IRQ_TIMER 45
> #define SPAREIRQ_IRQ_0 46
> #define SPAREIRQ_IRQ_1 47
> ```
> 
> and
> 
> ```c
>     PLL_USB_IRQ = 43, ///< Select PLL_USB's IRQ output
>     POWMAN_IRQ_POW = 44, ///< Select POWMAN's IRQ_POW output
>     POWMAN_IRQ_TIMER = 45, ///< Select POWMAN's IRQ_TIMER output
>     SPARE_IRQ_0 = 46, ///< Select SPARE IRQ 0
>     SPARE_IRQ_1 = 47, ///< Select SPARE IRQ 1
> ```
> 
> with the naming of `SPAREIRQ_IRQ_0` vs. `SPARE_IRQ_0` seeming to be a bit inconsistent? 🤔

yes, done.